src/main/java/com/mitchej123/hodgepodge/mixins/wandPedestalCV/MixinTileWandPedestal.java

```diff
@@ -0,0 +1,52 @@
+package com.mitchej123.hodgepodge.mixins.wandPedestalCV;
+
+import java.util.ArrayList;
+
+import net.minecraft.item.ItemStack;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.ChunkCoordinates;
+import net.minecraft.world.World;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import thaumcraft.api.TileThaumcraft;
+import thaumcraft.api.aspects.Aspect;
+import thaumcraft.api.aspects.AspectList;
+import thaumcraft.api.visnet.TileVisNode;
+import thaumcraft.api.visnet.VisNetHandler;
+import thaumcraft.common.items.wands.ItemWandCasting;
+import thaumcraft.common.tiles.TileNodeEnergized;
+import thaumcraft.common.tiles.TileWandPedestal;
+
+@Mixin(TileWandPedestal.class)
+public class MixinTileWandPedestal extends TileThaumcraft {
+    @Shadow
+    ArrayList<ChunkCoordinates> nodes = null;
+    @Shadow int counter = 0;
+
+    @Redirect(method ="Lthaumcraft/common/tiles/TileWandPedestal;updateEntity()V",
+        at=@At(value = "INVOKE",target="Lthaumcraft/common/items/wands/ItemWandCasting;getAspectsWithRoom(Lnet/minecraft/item/ItemStack;)Lthaumcraft/api/aspects/AspectList;"))
+    AspectList getAspectsWithRoomReplacement(ItemWandCasting wand, ItemStack wandstack)
+    {
+        AspectList as = wand.getAspectsWithRoom(wandstack);
+        if (as != null && as.size() > 0 && this.counter % 20 == 0) {
+            for(Aspect aspect : as.getAspects()) {
+                int drained = VisNetHandler.drainVis(worldObj, xCoord, yCoord, zCoord, aspect, 1);
```

Comment:
Conceptually, 100CV (centi-vis) is 1Vis.

Reply:
Huh, indeed. Ok, I'll have to accumulate it somwhere then